components/sup/protocols/event.proto

```diff
@@ -72,6 +72,48 @@ message ServiceStoppedEvent {
   ServiceMetadata service_metadata = 2;
 }
 
+message ServiceUpdateStartedEvent {
+  EventMetadata event_metadata = 1;
+  ServiceMetadata service_metadata = 2;
+  // Details of the service pkg to which 
+  // we're attempting to update.These
+  // are implemented as strings for now,
+  // but at some point we'll likely want
+  // to create a type for this set of
+  // attributes (PackageIdent)
+  string update_origin = 3;
+  string update_name = 4;
+  string update_version = 5;
+  string update_release = 6;
+}
+
+message ServiceUpdateCompleteEvent {
+  EventMetadata event_metadata = 1;
+  ServiceMetadata service_metadata = 2;
+  // Details of the service pkg to
+  // which we've successfully updated.
+  // These are implemented as strings for now,
+  // but at some point we'll likely want
+  // to create a type for this set of
+  // attributes (PackageIdent)
+  string update_origin = 3;
+  string update_name = 4;
+  string update_version = 5;
+  string update_release = 6;
```

Comment:
I think when this is sent, the identifier would be the same as what's in the `service_metadata`, so we probably don't need to send it.

Reply:
Also, it doesn't look like this message is used yet, so we should go ahead and remove it.

Reply:
@cm i think the identifier in the service_metadata corresponds to the currently running version of the service. the update fields are the values for the package to which the supervisor will update the service. lmk if i'm mistaken on this one.

Reply:
@gpeers it depends on when this event would be fired. If it is after the service has come back up (as I suspect / assume), then the updated identifier would indeed be the currently-running version, so they would be the same.

TL;DR - we're both right 😆

Reply:
@christophermaier nope, i insist... i read your comment too fast and didn't see that it referred to the service update complete event. thanks : )